tasks/dependencies.rb

```diff
@@ -32,8 +32,11 @@ def bundle_update_locked_multiplatform_task(task_name, dir)
     task task_name do
       Dir.chdir(dir) do
         Bundler.with_unbundled_env do
+          platform_list = "ruby x64-mingw32 x86-mingw32"
+          # chef-16 is now built using ruby31 with omnibus hence we need `x64-mingw-ucrt` platform ONLY for omnibus. The ruby version packaged inside chef continues to be ruby 2.7.5 (or as per omnibus_ovverrides.rb)
```

Comment:
`omnibus_ovverrides.rb` -> `omnibus_overrides.rb`

Reply:
Oops! How did I miss that. Thanks. Done.